pages/02.general/11.https-filtering/01.https-filtering-known-issues/docs.en.md

```diff
@@ -6,66 +6,66 @@ taxonomy:
 visible: true
 ---
 
-> Last update: 11 May 2017
+> Last update: 10 November 2017
 > Understanding this article may require from you the basic knowledge about encrypting, TLS protocol and HTTPS.
 
 First, look at this simple diagram that shows the general structure of HTTPS protocol:
 
 ![What is HTTPS filtering](https://cdn.adguard.com/public/Adguard/Blog/https/what_is_https_filtering.png)
 
-Adguard copies properties of the TLS-connection that your browser uses:
+AdGuard copies properties of the TLS-connection that your browser uses:
 
-* Adguard uses the same TLS version.
-* Adguard uses the same encrypting methods (ciphers) as your browser.
+* AdGuard uses the same TLS version.
+* AdGuard uses the same encrypting methods (ciphers) as your browser.
 
 Effectively, it means that if you use a modern, safe browser, it will take all known TLS problems into account and won’t attempt to use unsafe ciphers.
 
-**What does Adguard do when there are any doubts about the certificate’s validity?** In such cases, Adguard entirely ceases filtering of all connections to this domain and leaves the browser in charge of all decisions.
+**What does AdGuard do when there are any doubts about the certificate’s validity?** In such cases, AdGuard entirely ceases filtering of all connections to this domain and leaves the browser in charge of all decisions.
 
 ### Known issues
 
-HTTPS filtering in Adguard has its drawbacks. Almost all of them are scheduled to be eliminated in the nearest upcoming Adguard versions.  Moreover, we are working on a separate certificate validation library, and we want to make it open source.
+HTTPS filtering in AdGuard has its drawbacks. Almost all of them are scheduled to be eliminated in the nearest upcoming AdGuard versions. Moreover, we are working on a separate certificate validation library, and we want to make it open source.
 
 Below are listed all issues known to us and ETAs for the fixes.
 
 #### Phasing out SHA-1
 
-All modern browsers are discontinuing the support of SHA-1 certificates. This gradual process has begun in 2016 and should be finished in the 2017 Q1-Q2. For example, Chrome doesn’t trust SHA-1 certificates starting from v56, Microsoft Edge browser - since 9th of May 2017.
+All modern browsers are discontinuing the support of SHA-1 certificates. This gradual process has begun in 2016 and is almost finished by the end of 2017. For example, Chrome doesn’t trust SHA-1 certificates starting from v56, Microsoft Edge browser - since 9th of May 2017.
 
-Current stable Adguard versions do not trust leaf SHA-1 certificates but allow the use of the ‘intermediate’ ones.
+Current stable AdGuard versions do not trust either leaf or 'intermediate' SHA-1 certificates.
 
 **ETA:**
```

Comment:
Wouldn't it be better to remove this point as it's now fixed in all AG versions?